CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - MTL-2014: Include `spire-agent` in `csm-packages`
 - MTL-2025: Include `csm-node-identity` in `csm-packages`
 - CASM-3071/MTL-2021: Add ansible code for the csm-compute image
+- CASMNET-2085: Allow `enable-chn.yml` to be a no-op during Image Customization
```

Comment:
I think this is supposed to go under the `[Unreleased]` section above to avoid changing the contents of an earlier already released version. Then I think that `[Unreleased]` section becomes a new release version at some point.